js/src/carousel.js

```diff
@@ -152,6 +154,20 @@ class Carousel extends BaseComponent {
     }
     // End mod
 
+    // Boosted mod: if a play-pause button is present, set the button to play on mouseenter
+    if (this._element.nextElementSibling !== null) {
+      if (this._element.nextElementSibling.hasAttribute('data-bs-control') && this._element.nextElementSibling.classList.contains('pause')) {
+        this._element.nextElementSibling.classList.remove('pause')
+        this._element.nextElementSibling.classList.add('play')
+        this._element.nextElementSibling.setAttribute('title', 'Play Carousel')
+        this._element.nextElementSibling.querySelector('span.visually-hidden').innerHTML = 'Play Carousel'
+        this._isPaused = true
```

Comment:
IMO, this is dupe with following code

Reply:
This line is mandatory in order to prevent side effects between the play/pause button VS the already existing 'pause on hover' option.

Without this line, if you set the carousel to pause using the button and hover it with your cursor then the carousel will stay on pause on `mouseenter` and will restart to cycle on `mouseleave` but it will be stuck on the current slide.

**However it may be due to the bug I found on Bootstrap carousel but we will have to wait for their fix to check it.**